tests/testModelPsfMatch.py

```diff
@@ -7,6 +7,7 @@
 import lsst.ip.diffim as ipDiffim
 import lsst.log.utils as logUtils
 import lsst.meas.algorithms as measAlg
+from lsst.ip.diffim.modelPsfMatch import nextOddInteger
```

Comment:
If you changed to the binary operator in the other file, remember to change where the function is used here as well